packages/app-accounts/src/Editor.spec.js

```diff
@@ -0,0 +1,36 @@
+// Copyright 2017-2018 @polkadot/app-accounts authors & contributors
+// This software may be modified and distributed under the terms
+// of the ISC license. See the LICENSE file for details.
+
+import React from 'react';
+import { expect } from 'chai';
```

Comment:
Why are we using chai? Everywhere else just use the jest built-in expect, e.g (picking a file) https://github.com/polkadot-js/common/blob/master/packages/util/src/hex/fixLength.spec.js#L9